obc.service

```diff
@@ -4,7 +4,7 @@ After=network.target
 
 [Service]
 Type=simple
-ExecStart=/home/pi/.virtualenvs/cv/bin/python dir/obc.py
```

Comment:
We're actually gonna want to keep this file the way it is. If you do a `git fetch` followed by a `git checkout master -- obc.service` that will revert your branche's `obc.service` to what's on master

Reply:
You'll have to run a `git commit -a` after that as well

Reply:
Looks like the Exec path is still changed here. This is a bit of a tricky fix if you do it the way I told you to. But that's the safest way to revert a change

You can also just replace `ExecStart=/home/pi/.virtualenvs/cv/bin/python /mnt/c/Users/logan/OneDrive/Documents/Aero/onboard-computer/obc.py` 
with
`ExecStart=/home/pi/.virtualenvs/cv/bin/python dir/obc.py`